src/MdDocs.ApiReference.DemoProject/Grynwald.MdDocs.ApiReference.DemoProject.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Without this change, I was hitting a weird ambiguity error related to cref resolution of the `ObsoleteAttribute`. Some googling made it seem like this was a known transient-ish compiler bug. Updating to net6.0 fixed it; I presume that's ok given that other projects target net6.0 already.